pages/openid.js

```diff
@@ -0,0 +1,356 @@
+import OrganizationCard from 'components/org/organizationCard';
+import organizations from 'data/organizations';
+import Image from 'next/image';
+import React, { useEffect, useState } from 'react';
+import Link from 'next/link';
+import { Separator } from 'components/ui/separator';
+import DemoFlow from 'components/demo-flow';
+import { Button } from 'components/ui/button';
+import { QRCodeGenerator } from 'components/qrcode/qr-generator';
+import { apiGet, postRequest } from 'utils/request';
+import { dockUrl } from 'utils/constants';
+
+const credential = {
+  type: ['DockCredential'],
+  subject: {
+    nested: {
+      field: true,
+    },
+    inheritedField: 'test',
+  },
+  expirationDate: '2099-08-24T14:15:22Z',
+};
+
+const authProvider = {
+  url: 'https://creds.dock.io/claims',
+  scope: ['openid'],
+  clientId:
```

Comment:
Should `clientId` and `clientSecret` be in the .env file? Or is this just dummy data that will be discarded once we integrate it into the sales demo?

Reply:
> Should clientId and clientSecret be in the .env file

In this case it doesnt matter, because the creds claims server uses b64 encoded json for a client ID and the hash of that is the secret. It's not an actual client id/secret, but these values are "hijacked" to allow for defining custom form displays without using a database. If it was a typical oauth provider then you would want to put that in an env file for sure

> Or is this just dummy data that will be discarded once we integrate it into the sales demo?

Yeah this is all temporary anyway, it just defines a "name" field to request with a basic credential schema.